monitoring/monitoring_opencensus.go

```diff
@@ -0,0 +1,229 @@
+//go:build !skip_opencensus
```

Comment:
Whats the long term plan for this? its not easy to set build tags in istio release today. But not a big deal if its short lived

Reply:
Since OC is deprecated - it has to be short lived :-)

A separate PR will add an implementation based on prom client, and swap the default - I'm not sure if we should have an env variable to control or just switch, will get there - one step at a time, but I do not plan to have this in the release.

Reply:
Why prom? I thought we were moving to otel?

Reply:
Why not use otel SDK to allow exporters besides prometheus? Semantically they should be equivalent per their stated goal.